Web/Edubase.Web.UI/Controllers/AmalgamateMergeController.cs

```diff
@@ -141,8 +131,8 @@ public async Task<ActionResult> ProcessMergeEstablishmentsAsync(MergeEstablishme
             {
                 var estab2 =
                     await _establishmentReadService.GetAsync(model.Establishment2Urn.GetValueOrDefault(), User);
-                var hasErrors = ModelState.Keys.Where(k => k == "Establishment2Urn")
-                    .Select(k => ModelState[k].Errors).First().Select(e => e.ErrorMessage).Any();
+
+                var hasErrors = DoesModelStateHaveErrorsForKey("Establishment2Urn");
```

Comment:
Chatted to Scott about the tweaks made to this controller and happy with the changes